src/cclib/io/cjsonwriter.py

```diff
@@ -327,7 +327,8 @@ class NumpyAwareJSONEncoder(json.JSONEncoder):
     def default(self, obj):
         if isinstance(obj, np.ndarray):
             if obj.ndim == 1:
-                return obj.tolist()
+                nan_list = obj.tolist()
```

Comment:
No need to convert to a list, you can do `return obj[~np.isnan(obj)]`.

For reference you can read about [Boolean area indexing](http://docs.scipy.org/doc/numpy/reference/arrays.indexing.html#boolean-array-indexing)